Maddori.Apple/Maddori.Apple/Screen/View/HomeViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  HomeViewController.swift
+//  Maddori.Apple
+//
+//  Created by 이성민 on 2022/10/18.
+//
+
+import UIKit
+
+import SnapKit
+
+class HomeViewController: UIViewController {
+    
+    // MARK: - property
+    
+    private let firstLabel: KeywordLabel = {
+        let keyword = KeywordLabel()
+        keyword.keywordType = .defaultKeyword
+        keyword.text = "첫 djdkdkdk번째"
+        return keyword
+    }()
+    private let secondLabel: KeywordLabel = {
+        let keyword = KeywordLabel()
+        keyword.text = "첫 번째"
+        keyword.keywordType = .previewKeyword
+        return keyword
+    }()
+    
+    // MARK: - life cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configUI()
+        render()
+    }
+    
+    // MARK: - func
+    
+    private func configUI() {
+        view.backgroundColor = .black
+    }
+    
+    private func render() {
+        view.addSubview(firstLabel)
+        firstLabel.snp.makeConstraints {
+            $0.left.equalTo(view.safeAreaLayoutGuide).inset(24)
+            $0.top.equalTo(view.safeAreaLayoutGuide).inset(24)
```

Comment:
p2;

레이아웃을 잡을 땐 가급적 left right 의 사용보단 leading trailing의 사용을 권장합니다. 국가에 따라서 left right가 저희의 기준과 반대되는 경우가 있기 때문에 left -> leading, right -> trailing 으로 변경부탁드립니다 ^________^

```suggestion
            $0.leading.equalTo(view.safeAreaLayoutGuide).inset(24)
            $0.top.equalTo(view.safeAreaLayoutGuide).inset(24)
```